src/components/find-on-shelf/FindOnShelfPeriodicalDropdowns.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import { FC, useState } from "react";
+import ExpandMoreIcon from "@danskernesdigitalebibliotek/dpl-design-system/build/icons/collection/ExpandMore.svg";
+import { HoldingsForBibliographicalRecordV3 } from "../../core/fbs/model";
+import makePeriodicalEditionsFromHoldings from "../material/helper";
+import { groupObjectArrayByProperty } from "../../core/utils/helpers/general";
+
+export interface FindOnShelfPeriodicalDropdownProps {
+  manifestationsHoldings: HoldingsForBibliographicalRecordV3[];
+  setSelectedPeriodical: (selectedWeek: string | null) => void;
```

Comment:
Consider updating the signature here:

1. The value selected is probably not the week.
2. Do we need to allow null to be passed?